sechub-wrapper-secret-validator/src/test/java/com/mercedesbenz/sechub/wrapper/secret/validator/cli/SecretValidatorSpringBootTest.java

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.wrapper.secret.validator.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import com.mercedesbenz.sechub.wrapper.secret.validator.execution.SecretValidatorExecutionService;
+import com.mercedesbenz.sechub.wrapper.secret.validator.support.SarifImporterKeys;
+
+import de.jcup.sarif_2_1_0.model.Location;
+import de.jcup.sarif_2_1_0.model.Region;
+import de.jcup.sarif_2_1_0.model.Result;
+import de.jcup.sarif_2_1_0.model.Run;
+import de.jcup.sarif_2_1_0.model.SarifSchema210;
+
+@SpringBootTest
+@ExtendWith(SpringExtension.class)
+@EnableConfigurationProperties
+@TestPropertySource(properties = { "secret.validator.config-file=src/test/resources/config-test-files/valid-files/test-config.json",
+        "secret.validator.trust-all-certificates=false", "pds.job.result.file=src/test/resources/config-test-files/valid-files/test-result.txt" })
+@ActiveProfiles("test")
+class SecretValidatorSpringBootTest {
+
+    @Autowired
+    SecretValidatorExecutionService executionService;
+
+    @Test
+    void execution_service_with_correct_configuration_without_validation_categprizes_findings_with_default_configured() {
+        /* execute */
+        SarifSchema210 report = executionService.execute();
+
+        /* test */
+        Run run = report.getRuns().get(0);
+        for (Result finding : run.getResults()) {
+            Location location = finding.getLocations().get(0);
```

Comment:
Hmm... at the first glance I was a little bit confused - it took some seconds to understand, that really each finding shall have the same severity in this test cases and that really each finding must have no validation

What about introducing some assert methods here ?

e.g.
```
for (Result finding : run.getResults()) {
    assertFindingHasSecHubSeverity("medium");
    assertFindingIsNotValidated();
}
```
Also you could think about having 2 tests here and separate the assertions...